internal/webhooks/handler.go

```diff
@@ -29,13 +29,19 @@ const (
 	envOTELExporterOTLPEndpoint = "OTEL_EXPORTER_OTLP_ENDPOINT"
 	envOTELTracesExporter       = "OTEL_TRACES_EXPORTER"
 	envOTELResourceAttrs        = "OTEL_RESOURCE_ATTRIBUTES"
+	envJavaToolsOptions         = "JAVA_TOOL_OPTIONS"
 
-	exporterOTLP   = "otlp"
-	exporterJaeger = "jaeger-thrift-splunk"
+	volumeName        = "splunk-instrumentation"
+	initContainerName = "splunk-instrumentation"
+	javaJVMArgument   = " -javaagent:/splunk/splunk-otel-javaagent-all.jar"
+	exporterOTLP      = "otlp"
+	exporterJaeger    = "jaeger-thrift-splunk"
 
-	annotationConfInjectionEnabled = "o11y.splunk.com/inject-config"
-	annotationStatus               = "o11y.splunk.com/injection-status"
-	annotationReason               = "o11y.splunk.com/injection-reason"
+	annotationInstrumentation = "o11y.splunk.com/instrumentation"
+	annotationStatus          = "o11y.splunk.com/status"
+	annotationReason          = "o11y.splunk.com/reason"
```

Comment:
Why `inject(ion)` term removed? I think it's pretty important part defining annotations purpose .

What about:
`o11y.splunk.com/inject: config`
`o11y.splunk.com/inject: java-agent`
or
`o11y.splunk.com/inject-instrumentation: config`
`o11y.splunk.com/inject-instrumentation: java-agent`

Reply:
IMO feel `instrumentation` is clear enough and won't cause any ambiguity or confusion but not married to it. can change now or later.

Reply:
What is the attribute used in opentelemetry-collector-operator?

Reply:
Changed it to `o11y.splunk.com/inject-java: true`. This is similar to what community operator does. I think this addresses your concern and also allows users to inject multiple instrumentations for any possible edge cases.